### DISCUSSION TITLE
Allow setting a default number format

### DISCUSSION THREAD

**Discussion by jonahgreenthal** [getredash/redash#7139] [Ideas]
Redash supports a default date format and a default time format (in Settings > General). It would be good to support default number formats too.

My use case: The overwhelming majority of numbers I work with are integers, and I would prefer that they be formatted without thousands separators. For example, I want `12345`. However, Redash defaults to formatting with commas as thousands and showing two decimal places (e.g., `12,345.00`). I'd love to change the setting all at once rather than editing the visualization for every query, column-by-column.

**Comment by aesanchez** [comment]
Where you able to solve this? I'm interested in the same

**Comment by jgreenthal** [comment]
No — this is a feature request.

**Comment by ekonoval** [comment]
Same for me, I'm using it mostly to work with IDs, so it's very inconvenient to change number format all the time

**Comment by eradman** [comment]
This feature is missing in the UI, but it can bet set in the database

```sql
UPDATE organizations
SET settings = jsonb_set(settings::jsonb, '{settings,float_format}', '"0.0"', true);
UPDATE organizations
SET settings = jsonb_set(settings::jsonb, '{settings,integer_format}', '"0"', true);

SELECT jsonb_pretty(settings::jsonb)
FROM organizations;
```

**Reply by aesanchez** [reply]
Interesting, I'll give it a try

**Comment by sergerus** [comment]
Please be aware of the bug https://github.com/getredash/redash/issues/2666 
Integer numbers that greater then 2^53 can not be displayed correctly in JavaScript.
